### PR TITLE
[DPE-2377] Introduce github central uploader

### PR DESCRIPTION
**Pull request by welpaolo**

### CODE REVIEW

File: .github/workflows/release-tarball.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+name: Release CI
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  pull_request:
```
Comment (deusebio) [inline .github/workflows/release-tarball.yaml:8]: Can we remove this? I don't think we need to run release on any PR. Actually the release should really just be run either manually or scheduled

**Review by deusebio — APPROVED** [review]
Looks all good! Only one thing that I don't think we should run release on each PR

File: products.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        "lp-building-repo": "~data-platform/soss/+source/charmed-spark/+git/charmed-spark/",
+        "lp-building-branch-prefix": "lp-3.4",
+        "lp-consumer-key": "LP_CONSUMER_KEY_SPARK",
+        "lp-access-token": "LP_ACCESS_TOKEN_SPARK",
+        "lp-access-secret": "LP_SECRET_TOKEN_SPARK",
+        "artifactory-url": "ARTIFACTORY_DEPLOY_URL",
+        "artifactory-user": "ARTIFACTORY_USER",
+        "artifactory-token": "ARTIFACTORY_TOKEN",
+        "tarball-regex": "spark-*.tgz"
+    }
```
Comment (Mehdi-Bendriss) [inline products.json:15]:
```suggestion
    },
    {
        "name": "spark",
        "lp-building-project": "soss",
        "lp-releasing-project": "opensearch-releases",
        "lp-building-repo": "~data-platform/soss/+source/opensearch-build/+git/opensearch-build/",
        "lp-building-branch-prefix": "lp-2",
        "lp-consumer-key": "LP_CONSUMER_KEY_OPENSEARCH",
        "lp-access-token": "LP_ACCESS_TOKEN_OPENSEARCH",
        "lp-access-secret": "LP_SECRET_TOKEN_OPENSEARCH",
        "artifactory-url": "ARTIFACTORY_DEPLOY_URL",
        "artifactory-user": "ARTIFACTORY_USER",
        "artifactory-token": "ARTIFACTORY_TOKEN",
        "tarball-regex": "opensearch-*.tar.gz"
    }
```

**Review by Mehdi-Bendriss — APPROVED** [review]

File: products.json (hunk 2 above)
Comment (welpaolo) [reply products.json:15]: We need to test it, I will have a look at it

**Pull request merged**